### PR TITLE
fix(mojaloop/2809): individual transfer results is empty

### PR DESCRIPTION
**Pull request by vijayg10**
fix([mojaloop/2809](https://github.com/mojaloop/project/issues/2809)): individual transfer results is empty

### CODE REVIEW

**Review by kleyow — APPROVED** [review]

File: modules/api-svc/src/FSPIOPEventHandler/handlers/party-info-requested.js [hunk 1]
```diff
@@ -56,7 +56,11 @@ module.exports.handlePartyInfoRequestedDmEvt = async (
             bulkId: event.getKey(),
             content: {
                 transferId: event.getTransferId(),
-                partyResult: response,
```
Comment (kirgene) [inline modules/api-svc/src/FSPIOPEventHandler/handlers/party-info-requested.js:59]: I don't think there is a `body` in the response:
https://github.com/mojaloop/sdk-scheme-adapter/blob/16ccfe52f101d38daa397cf3312f33462bf8176c/src/OutboundServer/handlers.js#L467-L475

**Review by kirgene — CHANGES_REQUESTED** [review]

File: modules/api-svc/src/FSPIOPEventHandler/handlers/party-info-requested.js [hunk 2]
```diff
@@ -56,7 +56,11 @@ module.exports.handlePartyInfoRequestedDmEvt = async (
...
+                partyResult: {
+                    party: response.party?.body,
```
Comment (kirgene) [inline modules/api-svc/src/FSPIOPEventHandler/handlers/party-info-requested.js:60]: I don't think there is a `body` in the response:
https://github.com/mojaloop/sdk-scheme-adapter/blob/16ccfe52f101d38daa397cf3312f33462bf8176c/src/OutboundServer/handlers.js#L467-L475

**Review by kirgene — CHANGES_REQUESTED** [review]

File: modules/api-svc/src/FSPIOPEventHandler/handlers/party-info-requested.js (hunk 2 above)
Comment (vijayg10) [reply modules/api-svc/src/FSPIOPEventHandler/handlers/party-info-requested.js:60]: Hi @kirgene , there are body and headers in party. Yest that's weird, but its the response from model currently. So I made this transformation as workaround

**Review by kirgene — APPROVED** [review]

File: modules/api-svc/src/FSPIOPEventHandler/handlers/party-info-requested.js (hunk 2 above)
Comment (kleyow) [reply modules/api-svc/src/FSPIOPEventHandler/handlers/party-info-requested.js:60]: Thats a carry over from merging the mojaloop-connector into the sdk-scheme-adapter. The `headers` were needed in the synchronous responses so now the schema is typically `objectResponse: {body:body, headers:headers}`.

Comment (kleyow) [reply modules/api-svc/src/FSPIOPEventHandler/handlers/party-info-requested.js:60]: Confusing I know v - v but that was the way to keep everything standardized.

**Pull request merged**